sustainability_stock/models/account_move.py

```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    picking_id = fields.Many2one(comodel_name="stock.picking", string="Stock Transfer")
```

Comment:
I think we should have a more specific name here because it might conflict with other modules making links between account.move and stock.picking. freight_picking_id or carbon_picking_id